[2025-04-13T17:44:04.212+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: M3_GA_pipeline.fetch_from_postgresql manual__2025-04-13T17:44:00.995604+00:00 [queued]>
[2025-04-13T17:44:04.231+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: M3_GA_pipeline.fetch_from_postgresql manual__2025-04-13T17:44:00.995604+00:00 [queued]>
[2025-04-13T17:44:04.232+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-04-13T17:44:04.233+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2025-04-13T17:44:04.233+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-04-13T17:44:04.267+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): fetch_from_postgresql> on 2025-04-13 17:44:00.995604+00:00
[2025-04-13T17:44:04.273+0000] {standard_task_runner.py:52} INFO - Started process 247 to run task
[2025-04-13T17:44:04.279+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'M3_GA_pipeline', 'fetch_from_postgresql', 'manual__2025-04-13T17:44:00.995604+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_galuh_alifani_DAG.py', '--cfg-path', '/tmp/tmp05qd2u4q', '--error-file', '/tmp/tmp8melgdyl']
[2025-04-13T17:44:04.282+0000] {standard_task_runner.py:80} INFO - Job 30: Subtask fetch_from_postgresql
[2025-04-13T17:44:04.491+0000] {task_command.py:371} INFO - Running <TaskInstance: M3_GA_pipeline.fetch_from_postgresql manual__2025-04-13T17:44:00.995604+00:00 [running]> on host f42020b50842
[2025-04-13T17:44:04.702+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Galuh
AIRFLOW_CTX_DAG_ID=M3_GA_pipeline
AIRFLOW_CTX_TASK_ID=fetch_from_postgresql
AIRFLOW_CTX_EXECUTION_DATE=2025-04-13T17:44:00.995604+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-13T17:44:00.995604+00:00
[2025-04-13T17:44:10.071+0000] {logging_mixin.py:115} INFO - data has been loaded
[2025-04-13T17:44:10.148+0000] {python.py:173} INFO - Done. Returned value was: None
[2025-04-13T17:44:10.450+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=M3_GA_pipeline, task_id=fetch_from_postgresql, execution_date=20250413T174400, start_date=20250413T174404, end_date=20250413T174410
[2025-04-13T17:44:10.535+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-04-13T17:44:10.656+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
